package.json

```diff
@@ -23,11 +23,13 @@
   },
   "main": "./index.js",
   "bin": {
-    "ghost-inspector": "./bin/cli.js"
+    "ghost-inspector": "./bin/ghost-inspector"
```

Comment:
Hmm, I'm not sure I feel comfortable with this... Does this mean that NPM package now needs to download a prebuilt binary instead of just running the JavaScript code in the package?

Reply:
No no, I just renamed the `cli.js`  to `ghost-inspector`. The first line in the file is 

```
#!/usr/bin/env node
```

So it doesn't _need_ the `.js` extension. The reason this gets renamed is because if you trigger the yargs help text, instead of seeing `ghost-inspector` in the help text, you see this:

```
cli.js <command>

Commands:
  cli.js folder <command>        Manage folders within your Ghost Inspector account.
  cli.js organization <command>  Access organization details.
  cli.js suite-result <command>  Manage suite results within your Ghost Inspector account.
  cli.js suite <command>         Manage suites within your Ghost Inspector account.
  cli.js test-result <command>   Manage test results within your Ghost Inspector account.
  cli.js test-runner-ips         Fetch a list of test runner IP addresses by region.
  cli.js test <command>          Manage tests within your Ghost Inspector account.

...
```

Reply:
Good catch still, I forgot to add the renamed file.

Reply:
Ah, got it. Yea, the missing file is what threw me off.